src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java

```diff
@@ -0,0 +1,31 @@
+package eu.dissco.core.translator.component;
+
+import eu.dissco.core.translator.properties.WebClientProperties;
+import eu.dissco.core.translator.repository.SourceSystemRepository;
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+@Getter
```

Comment:
I think you just want a getter on source system name/id/endpoint, not on repository or webClientProperties

Reply:
comment still stands

Reply:
They are no longer exposed as internal attributes so didn't get a getter method